content/doc/developer/plugin-development/injecting-environment-variables.adoc

```diff
@@ -0,0 +1,67 @@
+---
+layout: developerguide
```

Comment:
IMO this would better be written as an archetype plugin in `jenkins-archetypes`, so you would have a complete _working_ (and tested!) example to play with or even start modifying.

Reply:
I think its overhead for such simple thing